include/glow/Graph/Graph.h

```diff
@@ -948,6 +948,11 @@ class Function final : public IRContainer {
       NodeValue scale, NodeValue mean, NodeValue var, unsigned_t channelIdx = 0,
       float epsilon = 1e-5, float momentum = 0.9);
 
```

Comment:
Could you please add a few comments here to describe the function?

Reply:
Added some description for the function.